lib/rapidsms/tests/scripted.py

```diff
@@ -192,6 +193,11 @@ def runParsedScript (self, cmds):
                 last_msg = txt
         finally:
             self.stopRouter()
+            try:
+                transaction.commit()
+            except Exception:
```

Comment:
yeah it should be fine to commit even when there's no changes, it's just a no op but it shouldn't raise an exception. i think it'd be better if we remove the try/except here otherwise it might confuse debugging